python/pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.black]
+line-length = 88
+target-version = ['py38']
+include = '\.pyi?$'
+extend-exclude = '''
+# A regex preceded with ^/ will apply only to files and directories
+# in the root of the project.
+^/foo.py  # exclude a file named foo.py in the root of the project (in addition to the defaults)
+'''
+
+[tool.coverage.run]
+source = ['pyspinw']
+
+[tool.github.info]
+organization = 'spinw'
+repo = 'spinw'
+
+[tool.poetry]
+name = "pyspinw"
+version = "0.0.2"
+description = "Python library for spin wave calculations"
+license = "BSD-3-Clause"
+authors = ["Duc Le", "Simon Ward", "Richard Waite"]
+readme = "../README.md"
+homepage = "https://spinw.org"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Topic :: Scientific/Engineering :: Physics",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 3 :: Only",
+]
+packages = [ { include = "pyspinw" } ]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<=3.12"
```

Comment:
So I think this will actually run on Python 3.7 (which is what the README says below)... should this be changed or do we not want to support 3.7 which will be end-of-lifed in a month and a half's time...

Reply:
I've dropped 3.7. It's EOL